synapse/handlers/federation.py

```diff
@@ -80,6 +81,24 @@
 logger = logging.getLogger(__name__)
 
 
+backfill_processing_before_timer = Histogram(
```

Comment:
Can you add a comment on these new metrics, mainly so we know that they were added to debug performance.